packages/ember-data/lib/system/model/attributes.js

```diff
@@ -24,14 +24,16 @@ Model.reopenClass({
     Example
 
     ```javascript
+    import Ember from 'ember';
+    import DS from 'ember-data';
 
-    App.Person = DS.Model.extend({
+    var Person = DS.Model.extend({
```

Comment:
I think you could still do 
`import Ember from 'ember'` 
then call the `Ember.get()`
@bmac c/d ?